javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-playlist-app/index.html

```diff
@@ -2,124 +2,80 @@
-                <polygon points="5 4 15 12 5 20 5 4"></polygon>
-                <line x1="19" y1="5" x2="19" y2="19"></line>
-              </svg>
-            </button>
-            <button
-              type="button"
-              aria-label="shuffle songs"
-              id="shuffle-btn"
-              class="btn-styles"
-            >
-              <svg
-                class="shuffle-btn"
-                xmlns="http://www.w3.org/2000/svg"
-                version="1.0"
-                viewBox="0 0 580.000000 386.000000"
-                preserveAspectRatio="xMidYMid meet"
-              >
-                <g
-                  transform="translate(0.000000,386.000000) scale(0.100000,-0.100000)"
-                  fill="#ff0000"
-                  stroke="none"
-                >
-                  <path
-                    d="M3616 2718 c-31 -44 -21 -69 59 -148 41 -41 75 -78 75 -82 0 -5 -124 -8 -275 -8 -253 0 -279 -2 -330 -21 -89 -33 -149 -97 -234 -248 -64 -116 -75 -130 -85 -114 -7 10 -39 67 -72 128 -68 126 -121 186 -202 223 l-57 27 -327 3 c-318 3 -328 2 -348 -18 -27 -27 -25 -66 5 -95 l24 -25 299 0 c326 0 340 -2 398 -57 29 -27 204 -330 204 -353 0 -6 -39 -82 -87 -168 -86 -155 -123 -203 -173 -226 -16 -7 -136 -13 -338 -16 l-314 -5 -19 -24 c-25 -30 -24 -66 1 -91 20 -20 30 -21 353 -18 l332 3 53 28 c79 42 130 99 196 222 33 61 65 118 72 128 10 16 20 4 80 -105 79 -143 135 -207 213 -245 l56 -28 288 -3 c158 -2 287 -6 287 -10 0 -4 -34 -41 -75 -82 -80 -79 -90 -104 -59 -148 11 -16 25 -22 51 -22 34 0 49 12 185 147 127 126 148 152 148 178 0 26 -22 52 -153 183 -97 96 -161 152 -174 152 -35 0 -73 -38 -73 -72 0 -24 15 -46 77 -109 l76 -79 -264 0 c-359 1 -342 -8 -482 249 l-89 161 83 151 c144 266 130 258 488 259 l264 0 -76 -79 c-62 -63 -77 -85 -77 -109 0 -34 38 -72 73 -72 13 0 77 56 174 152 131 131 153 157 153 183 0 26 -21 52 -148 178 -136 135 -151 147 -185 147 -26 0 -40 -6 -51 -22z"
-                  />
-                </g>
-              </svg>
-            </button>
+          <h1 class="fcc-title">freeCodeCamp</h1>
+          <div>
+            <img src="assets/decor-lines.svg" alt="decoration lines right" />
           </div>
-
-          <ul class="playlist-container"></ul>
+          <div>
+            <img src="assets/menu.svg" alt="menu button" />
+          </div>
+        </div>
+        <div class="player__content">
+          <div id="player__album-art">
+            <img
+              src="https://cdn.freecodecamp.org/MP3%20Player%20Project/Quincy%20Larson%20Album%20Art.jpg"
+              alt="song cover art"
+            />
+          </div>
+          <div class="player__display">
+            <div class="player__display--songArtist">
+              <p id="player__song-title">Cruising for a Musing</p>
+              <p id="player__song-artist">Quincy Larson</p>
+            </div>
+            <div class="player__buttons">
+              <button class="rewind">
+                <img src="assets/rewind.svg" alt="rewind button" />
```

Comment:
```suggestion
                <img src="assets/rewind.svg" alt="rewind" />
```
We do not need to add "button" to the alt text because screen readers will already announce that this is a button.